metadata/images.go

```diff
@@ -276,6 +276,30 @@ func (s *imageStore) Delete(ctx context.Context, name string, opts ...images.Del
 	})
 }
 
+func (s *imageStore) Count(ctx context.Context) (int64, error) {
+	namespace, err := namespaces.NamespaceRequired(ctx)
+	if err != nil {
+		return 0, err
+	}
+
+	var count int64
+	if err := view(ctx, s.db, func(tx *bolt.Tx) error {
+		bkt := getImagesBucket(tx, namespace)
+		if bkt == nil {
+			return nil // empty store
+		}
+
+		return bkt.ForEach(func(k, v []byte) error {
+			count++
+			return nil
+		})
```

Comment:
bucket has a [`Stats`](https://pkg.go.dev/go.etcd.io/bbolt#Bucket.Stats) method and its return has a [`KeyN`](https://pkg.go.dev/go.etcd.io/bbolt#BucketStats) field. Maybe we can use this directly and remove the foreach.

(`KeyN` is a `int` type)

Reply:
This is not giving me the expected value.
Looks like its 10x the value I'm expecting.